assets/components/organisms/button/button.scss

```diff
@@ -20,19 +20,19 @@
 
   &:hover,
   &.hover {
-    background: $tamarillo;
-    border-color: $tamarillo;
+    background: $dark-red;
+    border-color: $dark-red;
```

Comment:
We should create classes like `$primary-hover-color` and `$primary-active-color` maybe. That way we can reuse this value when needed.

Or maybe we should not extend `_button.scss` from Bootstrap but replace it and call their mixins:

```scss
@mixin button-variant($background, $border, $hover-background: darken($background, 7.5%), $hover-border: darken($border, 10%), $active-background: darken($background, 10%), $active-border: darken($border, 12.5%)) {
  ...
}
```

It seems to be covering a lot of what we override here, no?